servatrice/src/servatrice_database_interface.h

```diff
@@ -111,8 +112,12 @@ public slots:
                     LogMessage_TargetType targetType,
                     const int targetId,
                     const QString &targetName);
-    bool
-    changeUserPassword(const QString &user, const QString &oldPassword, const QString &newPassword, const bool &force);
+    bool changeUserPassword(const QString &user, const QString &password, bool passwordNeedsHash);
+    bool changeUserPassword(const QString &user,
+                            const QString &oldPassword,
+                            bool oldPasswordNeedsHash,
```

Comment:
Any reason oldPWNeedsHash and newPWNeedsHash? Shouldn't one be enough? (I could be wrong)

Reply:
there is no harm in having both even though the old one is not used right now

Reply:
If it's not used, i'm not sure why we need it exactly. Adds more complexity for no real value